tests/core/test_models.py

```diff
@@ -19,6 +19,33 @@
 _DEVICE = "cuda:0" if torch.cuda.is_available() else "cpu"
 
 
+def test_activation_functions_guassian_mlp():
+    activation_cfg_silu = omegaconf.OmegaConf.create(
```

Comment:
Looks good! One final comment, just add a test that when `use_silu=True` the given config is ignored. After that I'll run some full training tests and then merge. Thanks!

Reply:
@luisenp Done!